### PR TITLE
EDU-2367-EN-API-Overview-for-docs-portal

### PR DESCRIPTION
**Pull request by peeetry**
Related issue: [EDU-2367](https://aziontech.atlassian.net/browse/EDU-2367)

Changes:

* Introduction of API Overview.

[EDU-2367]: https://aziontech.atlassian.net/browse/EDU-2367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Get started with Azion API
+description: Learn more about oficial endpoints, methods, and API use cases of the Azion RESTful API.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:3]:
```suggestion
description: Learn more about official endpoints, methods, and API use cases of the Azion RESTful API.
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- **Real-Time Manager (RTM)** account.
+- **API token (personal token)**.
+
+:::tip
+Learn [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
+:::
+
+## Make API resquests
+ 
+Azion API documentation offers code examples to make requests with the terminal, using the curl tool, or to automate them with programming languages.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:32]:
```suggestion
Azion API documentation offers code examples to make requests with the terminal, using the curl command, or to automate them with programming languages. 
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+meta_tags: azion, rest api, restful api, azion restful api
+namespace: documentation_products_azion_api_overview
+permalink: /documentation/products/azion-api-overview/
+---
+
+This documentation will guide you on the first steps of using the **Azion API**.
+
+## API overview
+
+The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:13]:
```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. The complete Azion API collection is on Postman

<Button href="https://api.azion.com/" text="go to Azion API collection"></Button>

Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: Get started with Azion API
+description: Learn more about oficial endpoints, methods, and API use cases of the Azion RESTful API.
+meta_tags: azion, rest api, restful api, azion restful api
+namespace: documentation_products_azion_api_overview
+permalink: /documentation/products/azion-api-overview/
+---
+
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:8]:
```suggestion

import Button from '~/components/Button.astro'

```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx (hunk 3 above)
Comment (hannah-sk) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:13]: there's a link to the intro right after, but it seemed importante to clarify it's on Postman and add the button. we're using it in most docs!

**Review by hannah-sk — CHANGES_REQUESTED** [review]
one typo correction and a few suggestions!

**Review by MarianaAguilera — APPROVED** [review]
Approved with Hannah suggestions.

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+To run **Azion API** requests you need:
+
+- **Real-Time Manager (RTM)** account.
+- **API token (personal token)**.
+
+:::tip
+Learn [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
+:::
+
+## Make API resquests
```
Comment (peeetry) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:36]:
```suggestion
## Make API requests
```

**Comment by peeetry** [comment]
Closed to be reopened with the right GitHub files structure.

**Pull request closed without merging**